### ISSUE TITLE
FEAT: Inline edits in table editor directly from query result

### ISSUE THREAD

**Issue by tvarsis** [dbgate/dbgate#197]
Right now when you run a SELECT query, you are not able to edit the result directly in the output table (or at least I have not found a way to do it directly), but instead you will need to open up a new editor to be able to edit it. It would be a great improvement to the UX if it was possible to do inline edit directly from a query result as long as that query includes required id/primary keys etc that are needed for the inline action to work.

**Comment by janproch** [comment]
Good point. It will require some investigation, whether nodejs drivers offers required metadata about recordsets (original columns and tables). If so, this should be not to difficult to implement this feature

**Comment by Zenor27** [comment]
I approve. Did you investigate the feasibility since ?
